sopel/bot.py

```diff
@@ -587,32 +587,48 @@ def call_rule(
         sopel: 'SopelWrapper',
         trigger: Trigger,
     ) -> None:
+        nick = trigger.nick
+        context = trigger.sender
+        is_channel = context and not trigger.is_privmsg
+
         # rate limiting
         if not trigger.admin and not rule.is_unblockable():
-            if rule.is_rate_limited(trigger.nick):
+            if rule.is_rate_limited(nick):
```

Comment:
Since we've documented the Rule internals as subject to change without warning, any thoughts on taking this opportunity to rename this method as `rule.is_user_rate_limited` so it's parallel to its `get_user_rate_message` counterpart?

I'm not sure yet if there would be any utility in a replacement `rule.is_rate_limited` that could check multiple types at once, but that would be the logical choice.

Reply:
I totally agree. I'll go for the rename then.